pandas/io/formats/style.py

```diff
@@ -2230,7 +2230,7 @@ def set_caption(self, caption: str | tuple) -> Styler:
 
     def set_sticky(
         self,
-        axis: Axis = 0,
+        axis: Axis = "index",
```

Comment:
```suggestion
        axis: Axis = 0,
```

as previously mentioned pandas defaults Axis to integers